RequestManagementSystem/private/RequestValidator.py

```diff
@@ -19,32 +22,32 @@
 
     If you need to extend this one with your own specific checks consider:
 
-    * for adding Operation or Files required attributes use :addReqAttrsCheck: function::
+      * for adding Operation or Files required attributes use :any:`addReqAttrsCheck` function::
 
-    RequestValidator().addReqAttrsCheck( "FooOperation", operationAttrs = [ "Bar", "Buzz"], filesAttrs = [ "LFN" ] )
+          RequestValidator().addReqAttrsCheck( "FooOperation", operationAttrs = [ "Bar", "Buzz"], filesAttrs = [ "LFN" ] )
 
-    * for adding generic check define a new callable object ( function or functor ) which takes only one argument,
-      say for functor::
+      * for adding generic check define a new callable object ( function or functor ) which takes only one argument,
+        say for functor::
 
-    class MyValidator( RequestValidator ):
+          class MyValidator( RequestValidator ):
 
-      @staticmethod
-      def hasFoo( request ):
-        if not request.Foo:
-          return S_ERROR("Foo not set")
-        return S_OK()
+            @staticmethod
```

Comment:
http://diracsailer.readthedocs.io/en/fixdoc/CodeDocumentation/RequestManagementSystem/private/RequestValidator.html
It already gets syntax highlighting